aioshelly/rpc_device/device.py

```diff
@@ -538,3 +544,28 @@ def _parse_dynamic_components(self, components: dict[str, Any]) -> None:
                 for item in self._dynamic_components
             }
         )
+
+    async def _retrieve_blutrv_components(self, components: dict[str, Any]) -> None:
+        """Retrieve BLU TRV components."""
+        if self.model != MODEL_BLU_GATEWAY_GEN3:
+            return
+
+        if not self._config or not self._status:
+            raise ConnectionError
+
+        for component in components.get("components", []):
+            _key = component["key"].split(":")
+            if _key[0] == BLU_TRV_IDENTIFIER:
+                calls = [
+                    ("BluTrv.GetRemoteConfig", {"id": _key[1]}),
+                    ("BluTrv.GetRemoteStatus", {"id": _key[1]}),
+                ]
+                results = await self.call_rpc_multiple(calls)
+
+                cfg: dict[str, Any] = results[0]["config"]["trv:0"]
```

Comment:
What happens if there is more than 1 TRV?

Reply:
The id of the TRV changes (200,201,202) , so the key changes